### DISCUSSION TITLE
Unable to @dependabot in PR when using GitHub mobile

### DISCUSSION THREAD

**Discussion by DevDengChao** [community/community#73005] [Mobile] [answered]
### Select Topic Area

Bug

### Body

There is a screen record video, as you can see, there is only myself in the candidates, I cannot select dependabot, and the screen is flashing while I'm typing. 

If the video is not loaded, please watch it at https://youtube.com/shorts/aAi77ssKw5Q?feature=share .

<video src="https://github.com/community/community/assets/16363180/b98d1805-6c89-467a-b2d8-be2e7a2a8ea6" />

**Comment by Iron-Ham** [comment, marked as answer]
Hello @DevDengChao, 

Thank you for this report. This certainly seems like a bug. I'll file an internal issue to track this so that we can get this fixed. 


Thanks,
Hesham
